### ISSUE TITLE
Problem with the labels of the tools

### ISSUE THREAD

**Issue by JcToon** [2d-inc/support#170]
Using the tools menu, select the transformation tool, the tool label has appeared, but it has remained on the screen and does not disappear.
![label_problem](https://user-images.githubusercontent.com/45951634/50115821-c951e880-0248-11e9-8bdc-36ce43401cd8.jpg)

**Comment by neurowave** [comment]
Yeah, I've seen this one too! It usually hangs out on the top left of my screen, weird that it's in the middle of yours.
